curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.english.md

```diff
@@ -13,6 +13,8 @@ In ES5, we usually define a constructor function, and use the <code>new</code> k
 The class syntax simply replaces the constructor function creation:
 <blockquote>class SpaceShuttle {<br>&nbsp;&nbsp;constructor(targetPlanet){<br>&nbsp;&nbsp;&nbsp;&nbsp;this.targetPlanet = targetPlanet;<br>&nbsp;&nbsp;}<br>}<br>const zeus = new SpaceShuttle('Jupiter');</blockquote>
 Notice that the <code>class</code> keyword declares a new function, and a constructor was added, which would be invoked when <code>new</code> is called - to create a new object.
+#### Note
```

Comment:
Markdown doesnt work either, you'll have to use `<strong> </strong>` tag.

Reply:
I thought since the files were markdown files it would work. Figured there was some sort of conversion happening behind the scenes